### PR TITLE
Update/contact verification copy

### LINKED ISSUE #3802: Domains: tweak copy for domain contact verification notice
The notice shown when a domain registered with WordPress.com has unverified contact details mistakenly states that domains affected can be "lost forever". Further, it could use a link to the relevant support page.

Suggest changing the notice copy to the following:

> Urgent. Please verify your address through the verification email as soon as possible, or your domain may be suspended.

> (button)

> Use this button to resend the verification email. It contains a link to verify your address. [Learn more](https://en.support.wordpress.com/register-domain/#email-validation-and-verification).

The Learn More link points to: https://en.support.wordpress.com/register-domain/#email-validation-and-verification

This is re. some post-merge discussion in PR #3365.

cc @wensco @BandonRandon

### PR DESCRIPTION
**Pull request by BandonRandon**
Update copy for ICANN verification message. 

Fixes #3802

### CODE REVIEW

File: client/my-sites/upgrades/domain-management/components/icann-verification/icann-verification-card.jsx [hunk 1]
```diff
@@ -42,9 +41,12 @@ let IcannVerificationCard = React.createClass( {
 				<div className="icann-verification__explanation">
 					{ this.translate(
 						'Use this button to resend the verification email. It contains a link to verify your address. ' +
-						'To update your email address, go to {{a}}Contacts and Privacy{{/a}}.', {
+						'{{a}}Learn more{{/a}}.', {
 							components: {
-								a: <a href={ paths.domainManagementEditContactInfo( this.props.selectedSite.slug, this.props.selectedDomainName ) } />
+								a: <a
+								href="https://en.support.wordpress.com/register-domain/#email-validation-and-verification"
```
Comment (klimeryk) [inline client/my-sites/upgrades/domain-management/components/icann-verification/icann-verification-card.jsx]: It'd be great if you could add this URL to the [`support.js`](https://github.com/Automattic/wp-calypso/blob/master/client/lib/url/support.js) file and reference it from there. That way, if the URL ever changes, we'll need to update it in one place :)
You can see how to use this class this class [for example here](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/upgrades/domain-management/transfer/icann-verification.jsx#L14).

**Comment by klimeryk** [comment]
Please remember to [squash the commits](https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/) before merging :) (let me know if you need help with that)

**Comment by BandonRandon** [comment]
Edited link to include Support.js and rebased the branch.

**Comment by klimeryk** [comment]
Awesome! :shipit:

**Pull request merged**